index.html

```diff
@@ -23,25 +23,283 @@ <h1><br>I’m George<br>Glad to see you!</h1>
+                <img src="./images/frame10.svg" alt="Html Logo" />
+              </a>
+            </li>
+            <li>
+              <a href="#" class="frame11">
+                <img src="./images/frame11.svg" alt="CSS Logo" />
+              </a>
+            </li>
+            <li>
+              <div class="arrow arrow-hr">
+                <a href="#" class="frameworks">
+                  <img src="./images/Frameworks.svg" alt="Frameworks" />
+                </a>
+                <img src="./images/arrow.svg" alt="" />
+              </div>
+            </li>
+            <li>
+              <div class="arrow arrow-hr">
+                <a href="#" class="skills">
+                  <img src="./images/Skills.svg" alt="Skills" />
+                </a>
+                <img src="./images/arrow.svg" alt="" />
+              </div>
+            </li>
+          </ul>
+        </div>
+      </section>
+             <section>
+              <div class="contact-me">
+                <div class="contact-wrapper">
+                  <h2>Contact Me</h2>
+                  <p>
+                    If you have an application you are interested in developing a<br>
+                    feature that you need built or a project that needs coding. I’d
+                    love to help with it
+                  </p>
+                  <form action="https://formspree.io/f/meqngryy" method="post">
+                    <ul>
+                      <li>
+                        <input
+                          type="text"
+                          class="me"
+                          id="me"
+                          name="contactme"
+                          maxlength="30"
+                          placeholder="George D. Kruah"
+                          required
+                        />
+                      </li>
+                      <li>
+                        <input
+                          type="email"
+                          class="email"
+                          id="email"
+                          name="email"
+                          placeholder="dfizzkruah@gmail.com"
+                          required
+                        />
+                      </li>
+                      <li>
+                        <textarea
+                          name="message"
+                          class="message"
+                          maxlength="500"
+                          id="message"
+                          placeholder="Write your message here"
+                          required
+                        ></textarea>
+                      </li>
+                      <li>
+                        <button type="submit" name="get-in-touch">
+                          <strong>Get in Touch</strong>
+                        </button>
+                      </li>
+                    </ul>
+                  </form>
+                </div>
+              </div>
+            </section>
```

Comment:
Great job on implementing all the changes well except for the `background image` of the contact form.  Kindly use the right image for the background instead of the same image you used in the `hero` section.
| Your design | Figma design |
| --- | --- |
| ![image](https://user-images.githubusercontent.com/24703171/155314124-50fbfb89-3752-45e4-82f9-9b416d00ea9a.png) | ![Template 2-Mobile-Main (1)](https://user-images.githubusercontent.com/24703171/155314167-88fbb205-256e-4c71-b565-9faacd4d4f41.png) |

Reply:
@gemmen29 .
Kindly review changes made to the contact form background image as was stated to be sure if there's anything left before it for final review. 

Thanks for the feedback.

Reply:
Still not match the Figma design

Reply:
@gemmen29 . I made changes to the background -image.
Can you please indicate what is required that's not matching the Figma design?

Reply:
This is the design now in the mobile version
![image](https://user-images.githubusercontent.com/24703171/155324539-92da6e3c-127c-4a30-b4f3-df7842dabf5f.png)
the background is not the same as the Figma design and inputs are outside the background on the right

Reply:
![Screenshot (303)](https://user-images.githubusercontent.com/18019955/155327315-ca153433-7138-4915-954c-eb52ce7348a3.png)

That's the previous version. I have made changes to the screenshot sent.
Please see attached file of the changes made

Reply:
@gemmen29 
Kindly see changes made to the background image of the contact form and also change to the mobile version as was indicated.